core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20180221_drop_project_group_table.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="dummy" author="mbemis" id="drop-project-group-table-1">
+        <sql stripComments="true">
+            drop table BILLING_PROJECT_GROUP
```

Comment:
this table is no longer used as of sam phase 3. we could have deleted it long ago but didn't.

it's only semi-related to the PR but i realized we still had FKs in place while testing this code against a rawls dev DB clone